### ISSUE TITLE
Please create a tag for Coq 8.20 in Coq Platform 2025.01

### ISSUE THREAD

**Issue by MSoegtropIMC** [math-comp/real-closed#66]
The Coq team released Coq `8.20.0` on September 3rd, 2024.
The corresponding Coq Platform release `2025.01` should be released before **Jan 31st, 2025**.
It can be delayed in case of difficulties until Feb 28th, 2025, but this should be an exception.

This issue is to inform you that to our (possibly a few days old) best knowledge the latest released version of your project (2.0.1) **does not work** with Coq `8.20.0`.
We tried to remove version restrictions in opam files and possibly make or configure files, but this did not suffice.

Please note that in Coq Platform CI (unlike Coq CI) we test only released / tagged versions. Coq CI appears to test this project, but has some special handling for your project which makes it difficult to retrieve the commit it tests for your project.

Could you please create a tag and opam package, or communicate us any existing tag that works with Coq branch v8.20, **preferably before Dec 31st, 2024**?
In case we might have to delay the Coq Platform release cause of issues with your project, we would prefer to be informed about the situation as early as possible.

In case the tag and opam package are available before Dec 8th, 2024, it will be included in an early Coq Platform beta release of the for Coq 8.20.0.

The working branch of Coq Platform, can be found here [main](https://github.com/coq/platform/tree/main).
It contains package pick [`~8.20~2025.01`](https://github.com/coq/platform/tree/main/package_picks/package-pick-8.20~2025.01.sh) which already supports Coq version `8.20.0` and contains already working (possibly patched / commit pinned) Coq Platform packages.

Please **don't** close this issue, even after creating the new tag and/or opam package.
We will close the issue after updating Coq Platform.
This is a measure of 'double book keeping' in order to avoid that we ship the wrong version.

In any case, Coq Platform won't be released before this issue is closed!

Thanks!

P.S.: this issue has been created automatically based on CI status.

CC: https://github.com/coq/platform/issues/449

**Comment by MSoegtropIMC** [comment]
@CohenCyril : I have seen that you created a new tag 2.0.2. Can you please confirm here that this is the tag you would like to see in Coq Platform 2025.01?

**Comment by CohenCyril** [comment]
@MSoegtropIMC yes please.

**Comment by MSoegtropIMC** [comment]
@CohenCyril : unfortunately 2.0.2 does not compile when mathcomp-analysis is installed - the well known log path conflict issue:
```
File "./theories/qe_rcf.v", line 3, characters 0-77:
Error: Required library path matches several files in path (found
/Users/msoegtrop/.opam-current+working-tests/CP.2024.10.1~8.20~2025.01/lib/coq/user-contrib/mathcomp/analysis/homotopy_theory/path.vo and
/Users/msoegtrop/.opam-current+working-tests/CP.2024.10.1~8.20~2025.01/lib/coq/user-contrib/mathcomp/ssreflect/path.vo).
```
Can you please fix this (by giving a more specific require name)?

**Comment by CohenCyril** [comment]
Dear @MSoegtropIMC as @palmskog [said](https://github.com/coq-community/reglang/issues/75#issuecomment-2551202673), the proper solution is to make a new release of mathcomp analysis ASAP.

**Comment by MSoegtropIMC** [comment]
@CohenCyril : I got confirmation in https://github.com/math-comp/analysis/issues/1429 and via Zulip that this is planned to be fixed by end of this year, so it is fine to wait.

**Comment by affeldt-aist** [comment]
Spreading the news: we made a new release of MathComp-Analysis (1.8.0) a few hours ago.

**Comment by MSoegtropIMC** [comment]
I updated Coq Platform to use coq-mathcomp-real-closed version 2.0.2 via https://github.com/coq/platform/pull/454 => closing.
